src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -226,6 +247,9 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
     margin-top: 0.5rem;
     padding-top: 0;
   }
+  #notation-text {
+    padding-top: 15px;
```

Comment:
Do you need the padding as well as the margin? (The margin uses class helper "mt-2" in the template.) You may be able to use just one of these.

PS We try to use the class helpers as much as possible. In this case the closest one would be "pt-4" (16px). Is that close enough or does UI/UX want exactly 15px?

Reply:
Nice fix!